.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: generic 
+install:
+    - gem install mdl
+script:
+    - mdl .
```

Comment:
can we disable emails as well?

Reply:
... that is a very good idea.

Reply:
honestly the emails should be disabled by default... silly travis